### PR TITLE
[OCF-63] runtime/action-python-2: Switch to CentOS 7

### PR DESCRIPTION
**Pull request by goldmann**
https://issues.jboss.org/browse/OCF-63

### CODE REVIEW

**Comment by goldmann** [comment]
I'm not sure why the test failed, but I do see similar failures in my local environment, even with the original image.

File: docker/runtimes/action-python-2/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,28 @@
-FROM openwhisk/python2action:1.0.0
+FROM jboss/base:latest
```
Comment (jcrossley3) [inline docker/runtimes/action-python-2/Dockerfile]: I don't love latest here

**Review by jcrossley3 — CHANGES_REQUESTED** [review]

**Comment by goldmann** [comment]
@jcrossley3 This one is ready

File: docker/runtimes/action-python-2/Dockerfile [hunk 2]
```diff
@@ -1,8 +1,29 @@
-FROM openwhisk/python2action:1.0.0
+# Latest image from Mar 22nd
+FROM jboss/base@sha256:39bcf23f34ca58db0769121674d2a82aa4ea2ae9c956e280cb0ba1ef64c68b51
```
Comment (jcrossley3) [inline docker/runtimes/action-python-2/Dockerfile:2]: Now *THAT* looks pretty specific! Thanks!

**Review by jcrossley3 — APPROVED** [review]

**Pull request merged**